astro/src/content/quickstarts/quickstart-javascript-express-api.mdx

```diff
@@ -0,0 +1,357 @@
+--header 'Content-Type: application/json' \
+--data-raw '{
+  "loginId": "teller@example.com",
+  "password": "password",
+  "applicationId": "e9fdb985-9173-4e01-9d73-ac2d60d1dc8e"
+}'
+```
+
+Copy the `token` from the response, which should look like this:
+
+```json
+{
+  "token": "eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCIsImtpZCI6InVOYl9iQzFySHZZTnZMc285VzRkOEprZkxLWSJ9.eyJhdWQiOiJlOWZkYjk4NS05MTczLTRlMDEtOWQ3My1hYzJkNjBkMWRjOGUiLCJleHAiOjE2ODkzNTMwNTksImlhdCI6MTY4OTM1Mjk5OSwiaXNzIjoiaHR0cDovL2xvY2FsaG9zdDo5MDExIiwic3ViIjoiMDAwMDAwMDAtMDAwMC0wMDAwLTAwMDAtMTExMTExMTExMTExIiwianRpIjoiY2MzNWNiYjUtYzQzYy00OTRjLThmZjMtOGE4YWI1NTI0M2FjIiwiYXV0aGVudGljYXRpb25UeXBlIjoiUEFTU1dPUkQiLCJlbWFpbCI6InRlbGxlckBleGFtcGxlLmNvbSIsImVtYWlsX3ZlcmlmaWVkIjp0cnVlLCJhcHBsaWNhdGlvbklkIjoiZTlmZGI5ODUtOTE3My00ZTAxLTlkNzMtYWMyZDYwZDFkYzhlIiwicm9sZXMiOlsiY3VzdG9tZXIiLCJ0ZWxsZXIiXSwiYXV0aF90aW1lIjoxNjg5MzUyOTk5LCJ0aWQiOiJkN2QwOTUxMy1hM2Y1LTQwMWMtOTY4NS0zNGFiNmM1NTI0NTMifQ.WLzI9hSsCDn3ZoHKA9gaifkd6ASjT03JUmROGFZaezz9xfVbO3quJXEpUpI3poLozYxVcj2hrxKpNT9b7Sp16CUahev5tM0-4_FaYlmUEoMZBKo2JRSH8hg-qVDvnpeu8nL6FXxJII0IK4FNVwrQVFmAz99ZCf7m5xruQSziXPrfDYSU-3OZJ3SRuvD8bMopSiyRvZLx6YjWfBsvGSmMXeh_8vHG5fVkq5w1IkaDdugHnivtJIivHuCfl38kQBgw9rAqJLJoKRHHW0Ha7vHIcS6OCWWMDIIVspLyQNcLC16pL9Nss_5v9HMofow1OvQ9sUSMrbbkipjKq2peSjG7qA",
+  "tokenExpirationInstant": 1689353059670,
+  "user": {
+      ...
+  }
+}
+```
+
+### Make The Request
+
+<Aside type="note">
+The code is set up to extract the token from either a cookie or the Authorization header so depending on your preference you can replace `--cookie 'app.at=<your_token>'` with `--header 'Authorization: Bearer <your_token>'` when making requests to the API.
+
+If you use a cookie, make sure you store it in a secure, HTTPOnly cookie to avoid exfiltration attacks. See [Storing OAuth Tokens](/articles/oauth/oauth-token-storage) for more information.
+</Aside>
+
+Make a request to `/make-change` with a query parameter `total=1.02`. Use the `token` as the `app.at` cookie.
+
+```shell
+curl --cookie 'app.at=<your_token>' 'http://localhost:3000/make-change?total=1.02'
+```
+
+Alternatively you can make the same request by passing your token in the `Authorization` header.
+
+```shell
+curl --location 'http://localhost:3000/make-change?total=1.02' \
+--header 'Authorization: Bearer <your_token>'
+```
+
+Your response should look like this:
+
+```json
+{
+  "total":1.02,
+  "nickels":20,
+  "pennies":2
+}
+```
+
+You were successfully authorized. You can try making the request without the JWT or with a different string rather than a valid token, and see that you are denied access.
+
+Next, call the `/panic` endpoint because you are in trouble!
+
+```shell
+curl --location --request POST 'http://localhost:3000/panic' --cookie 'app.at=<your_token>'
+```
+
+This is a POST not a GET because you want all your emergency calls to be non-idempotent.
+
+Your response should look like this:
+
+```json
+{
```

Comment:
my response did not contain json, just the message.

Reply:
@mark-robustelli the change which returns json is in this open PR https://github.com/FusionAuth/fusionauth-quickstart-javascript-express-api/pull/1. Did you use that code when testing.